node_test.go

```diff
@@ -0,0 +1,96 @@
+package iavl
+
+import (
+	"bytes"
+	"math/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNode_aminoSize(t *testing.T) {
+	node := &Node{
+		key:       randBytes(10),
+		value:     randBytes(10),
+		version:   1,
+		height:    0,
+		size:      100,
+		hash:      randBytes(20),
+		leftHash:  randBytes(20),
+		leftNode:  nil,
+		rightHash: randBytes(20),
+		rightNode: nil,
+		persisted: false,
+	}
+
+	// leaf node
+	var buf bytes.Buffer
+	err := node.writeBytes(&buf)
+	require.NoError(t, err)
+	require.Equal(t, buf.Len(), node.aminoSize())
```

Comment:
Here you're testing`node.writeBytes` returns exactly `node.aminoSize()`, but we want to test that `particular node.aminoSize() == 10` for example

Reply:
changed.